src/main/java/com/eva/model/person/applicant/InterviewDate.java

```diff
@@ -61,6 +54,6 @@ public boolean isCompleted() {
 
     @Override
     public String toString() {
-        return interviewDate.toString();
+        return DateUtil.dateToString(interviewDate);
```

Comment:
May wanna change the way interview dates are represented. (I think this is for Nikhila?)